customer-config/scan-and-go/sng.settings.v1.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Hmm I know we talked about checkout engine url in ccc, but didn't we agree to not do it? Just double checking. Can you get an approach from @sasjo please?

Reply:
Yes, the final decision was to use ccc.